### PR TITLE
Adds blocks navigation

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

Adds blocks navigation that also support layout and nested blocks 🎉 

There was a minor focus issue with the PR that I thought I had completed before. 
We solved it with the help of @distantnative, thank you so much. 
I hope it's an acceptable PR.

blocks-navigation

## Ready?

- [ ] Added unit tests for fixed bug/feature
- [x] Passing all unit tests
- [x] Fixed code style issues with CS fixer and `composer fix`
- [x] Added in-code documentation (if needed)

### CODE REVIEW

**Comment by afbora** [comment]
By the way, if there is a point you see missing in PR, it is enough to give feedback, I can be interested 👍

**Comment by bastianallgeier** [comment]
I love the navigation arrows. The PR is well done. I sat at this briefly today and was wondering about the position of the arrows. In general they should get a bit more padding to make it easier to click them. But I'm also not entirely sure yet if they should be in the header.

**Comment by afbora** [comment]
Yes, a little more padding can be given. But I couldn't imagine a position other than the header. The common area of the blocks is the header and I think this is the ideal place I can think of. If you don't have a very different navigation style in mind, we can enhance it in the future versions.

**Comment by afbora** [comment]
screen-capture-584-Mægazine-localhost

I've improved the look a bit more.

File: panel/src/components/Blocks/Block.vue [hunk 1]
```diff
@@ -220,4 +256,8 @@ export default {
 .k-block-container[data-hidden] .k-block {
   opacity: .25;
 }
+.k-drawer-options .k-button[data-disabled] {
+  vertical-align: middle;
+  display: inline-grid;
+}
```
Comment (afbora) [inline panel/src/components/Blocks/Block.vue:262]: When disabled, the button was converted to span and the appearance was broken. Fixes this issue.

Comment (bastianallgeier) [reply panel/src/components/Blocks/Block.vue:262]: I was tripping over the same problem. Great catch!

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**